README.md

```diff
@@ -1,4 +1,53 @@
-Otter Browser
-=============
+# ![Otter Browser logo](resources/icons/otter-browser-64.png) Otter Browser
+
+
+***A browser controlled by the user, not vice-versa***
+
+[**Otter Browser**](https://otter-browser.org/) aims to recreate the best aspects of **Opera 12** and to revive its spirit. We are working on providing the powerful features power users want while keeping the browser fast and lightweight. We also learned from History and decided to release the browser under the GNU GPL v3.
+
+[![SourceForge](https://img.shields.io/sourceforge/dt/otter-browser.svg)](https://sourceforge.net/projects/otter-browser/files/)
+
+[![Screenshot](https://otter-browser.org/screenshots/1.png)](https://otter-browser.org/screenshots/)
+
+Planned features are listed in the *TODO* file. Details on already implemented features are available in the *CHANGELOG* file. If you have an idea that has not yet been proposed or refused, feel free to [open a new issue](https://github.com/OtterBrowser/otter-browser/issues/new).
+
+## Install
+
+You can either compile Otter Browser from source or use pre-compiled binaries.
+
+### From source
+
+To build Otter, you will need the following depedencies: Qt (at least 5.4.0), Sonnet, and CMake (3.1.0 or newer). At the root of the directory were the source code is stored, execute these commands:
+
+    mkdir build
+    cd build
+    cmake ../
+    make
+    make install
+
+Detailed instructions are available in the *INSTALL* file at the root of the repository.
+
+### Under Linux and *BSD
+
+Linux users can use the official AppImage version available on [SourceForge](https://sourceforge.net/projects/otter-browser/files/). It is a single executable file that doesnt need any dependencies to be installed. The AppImage version should run under any system installed after 2012. The browser is also available in a the repositories of a wide range of Linux distributions and *BSD systems. Read more on [the dedicated wiki page](https://github.com/OtterBrowser/otter-browser/wiki/Packages).
```

Comment:
AppImage has some runtime dependencies, right now it's OpenSSL 1.0.x (but not 1.1.x, they broke ABI in minor release...) and GStreamer 1.x and its codecs. OpenSSL (or LibreSSL) might be added to that bundle someday but not GStreamer (since the only legal way to allow support for proprietary codecs is to use system wide version of that library).